docs/dev_overview.md

```diff
@@ -1,11 +1,10 @@
 # µRaiden
 
 µRaiden components overview.
-! Some parts might be outdated.
 
  * [HTTP Request and Response Headers](#http-request-and-response-headers)
  * [Exceptions](#exceptions)
- * [Off-Chain Messages](#off-chain-messages)
+ * [Off-Chain Micropayment Sequence](#off-chain-micropayment-sequence)
```

Comment:
All changes to markdown-files that are not `README.md`- files should be updated in the corresponding `.rst` Version in the `doc-update` branch.
Now it's just up to who does it, so I can do it manually. But please do so for future changes.
We should get #345 merged ASAP to avoid further confusion.

Reply:
since this is a small change I suggest to update `doc-update` branch directly

Reply:
I will put all changes from the dev-overview in it's corresponding rst file manually.
Please create a PR against `doc-update` from now on for everything regarding documentation.